### PR TITLE
Light refactor, questions to address, and alt-text

### PR DESCRIPTION
**Pull request by plaindocs**
Very much draft status.

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -16,9 +16,15 @@ A Locust test is essentially just a Python program making requests to the system
...
+This user will make an HTTP request to ``/hello``, then to ``/world``, and then repeat. By default, Locust repeats tasks indefinitely.
 
-Change ``/hello`` and ``/world`` to some actual paths on the website/service you want to test, put the code in a file named ``locustfile.py`` in your current directory and then run ``locust``:
+.. TODO suprisingly hard to find the source of this repetition, needs link / definition
+
+For a full explanation and a more realistic example see :ref:`writing-a-locustfile`.
+
+Change ``/hello`` and ``/world`` in the previous code block to two endpoints on the service you want to test, put the code in a file named ``locustfile.py`` in your current directory and then run ``locust``:
+
+.. TODO would it make sense to include a few lines of python you can run to serve two endpoints to test this with? Or do we assume that everyone who gets here can do that if they need to. `python3 -m http.server 9000` and empty `hello` and `world` files does the trick, not sure if you can make that a oneliner`
```
Comment (cyberw) [inline docs/quickstart.rst:27]: I think having such an example makes a ton of sense! not sure what the best oneliner is...

File: docs/quickstart.rst [hunk 2]
```diff
@@ -27,22 +33,34 @@ Change ``/hello`` and ``/world`` to some actual paths on the website/service you
...
+
+Open the :ref:`web interface <interface>` to view the test results, or restart locust with the ``--headless`` parameter to view them on the :ref:`command line <headless>`.
+
+.. _interface:
+
 Locust's web interface
 ======================
 
-Open http://localhost:8089
+Locust's web interface is available by default on http://localhost:8089 . Specify some basic test parameters and a host to test, and once it's started, get reports on the running tests.
```
Comment (cyberw) [inline docs/quickstart.rst:47]: "Specify some basic test parameters and a host to test" is a bit vague? This is a quickstart guide so I'd like it to be a little more specific :)

File: docs/quickstart.rst [hunk 3]
```diff
@@ -27,22 +33,34 @@ Change ``/hello`` and ``/world`` to some actual paths on the website/service you
...
 .. image:: images/webui-running-statistics.png
+    :alt: Screenshot of the Locust web interface statistics page
 
 | Under the *Charts* tab you'll find things like requests per second (RPS), response times and number of running users:
 
 .. image:: images/bottlenecked_server.png
+    :alt: Screenshot of the Locust web interface charts page
+
+
+.. TODO can we at least point folks to external references on interpreting these graphs?
```
Comment (cyberw) [inline docs/quickstart.rst:63]: I'm writing some blog articles (https://medium.com/locust-cloud/load-testing-part-1-0f2332aa4a07, https://medium.com/locust-cloud/smoke-stress-spike-soak-and-recovery-5-essential-load-test-profiles-4606168d9c1a), we might want to link them somewhere :) 

I might write an entire post about interpreting results... but maybe leave this for later.

File: docs/quickstart.rst [hunk 4]
```diff
@@ -81,6 +101,8 @@ See :ref:`running-without-web-ui` for more details.
 More options
 ============
 
+.. TODO I suspect talking about distributed locust is overkill for the first test, and can safely bump it to writing a locustfile?
```
Comment (cyberw) [inline docs/quickstart.rst:104]: 👍 Makes sense!

File: docs/quickstart.rst [hunk 5]
```diff
@@ -27,22 +33,34 @@ Change ``/hello`` and ``/world`` to some actual paths on the website/service you
     [2021-07-24 09:58:46,215] .../INFO/locust.main: Starting web interface at http://0.0.0.0:8089
     [2021-07-24 09:58:46,285] .../INFO/locust.main: Starting Locust |version|
 
+Congratulations, you've run a Locust test!
+
+.. TODO you can't run a test first, and open the web interface to connect to that right? You either need to run it from the web interface, or specify the details and run in headless
```
[the remark this answers is not in the document]
Comment (plaindocs) [reply docs/quickstart.rst:38]: Agreed :-D

**Comment by plaindocs** [comment]
60 day? WOW. Let me get this back on track.

**Comment by cyberw** [comment]
🚀 I wont be able to feedback in the coming weeks though, just so you know :)

**Pull request closed without merging**